packages/react-combobox/src/components/ComboButton/ComboButton.test.tsx

```diff
@@ -8,6 +8,8 @@ describe('ComboButton', () => {
     Component: ComboButton,
     displayName: 'ComboButton',
     primarySlot: 'content',
+    // don't test deprecated className export on new components
+    disabledTests: ['component-has-static-classname-exported'],
```

Comment:
@TristanWatanabe We should bring up removing this test at the next v-build team meeting.